src/api/todos.ts

```diff
@@ -0,0 +1,8 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+// Add more methods here
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)